.github/workflows/ci.yaml

```diff
@@ -0,0 +1,49 @@
+name: Continuous Integration
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  linux-mono:
+    name: Linux (mono)
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Clone Repository
+        uses: actions/checkout@v2
+
+      - name: Prepare Environment
+        run: |
+          . mod.config;
+          awk '/\r$$/ { exit(1); }' mod.config || (printf "Invalid mod.config format. File must be saved using unix-style (LF, not CRLF or CR) line endings.\n"; exit 1);
+
+      - name: Check Code
+        run: |
+          make check
+
+      - name: Check Mod
+        run: |
+          sudo apt-get install lua5.1
+          make check-scripts
+          make test
+
+  windows:
+    name: Windows (Framework 4.7)
+    runs-on: windows-2019
+
+    steps:
+      - name: Clone Repository
+        uses: actions/checkout@v2
+
+      - name: Check Code
+        shell: powershell
+        run: |
+          .\make.ps1 check
```

Comment:
You may want https://github.com/OpenRA/OpenRA/pull/18968 here.

Reply:
Is it enough to do it on Linux only? Those are probably different VMs.

Reply:
🤦 Fixed, thanks.